odra-test/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "odra-test"
+edition = "2021"
+version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[dependencies]
+odra-core = { path = "../core" }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
```

Comment:
`odra-test` is not intended to be used in `wasm32`, this can be removed.
Instead you can add in `lib.rs`:
```rust
#[cfg(target_arch = "wasm32")]
compilation_error("odra-test in wasm32 target")
```